src/helpers.js

```diff
@@ -43,6 +43,12 @@ let setup = (includeDev) => {
 
     console.log(command);
     console.log();
+
+    /* Check if node modules exist and then backup */
+    let nodeModulesExist = fs.existsSync('node_modules');
+    if (nodeModulesExist) syncExec('cp -r node_modules node_modules_bak', {stdio: [0, 1, 2]});
```

Comment:
This will fail on windows (`'cp' is not recognized as an internal or external command,
operable program or batch file.`). [I suggest using node native renaming](https://nodejs.org/api/fs.html#fs_fs_renamesync_oldpath_newpath).